src/core/BufferAttribute.d.ts

```diff
@@ -151,7 +151,7 @@ export class Float64Attribute extends BufferAttribute {
 export class Int8BufferAttribute extends BufferAttribute {
 
 	constructor(
-		array: Iterable<number> | ArrayLike<number> | ArrayBuffer,
+		array: Iterable<number> | ArrayLike<number> | ArrayBuffer | number,
```

Comment:
These `array` are passed to typed array constructors, which accept a number as the length of the array to be constructed.